pkgs/development/compilers/ghc/8.4.2.nix

```diff
@@ -79,6 +81,8 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "doc" ];
 
+  patches = [./8.4.2-hsc2hs-cross-build.patch];
```

Comment:
@Ericson2314 Should this be unconditional like this? It's harmless on the normal build, but will require a rebuild of GHC 8.4.2.

Reply:
Yeah I agree. Simple is better.